Datetime bug with dealing with undefined values (#106)
datajoint/datajoint-labbook: CHANGELOG.md

- CSS for making the primary keys look disabled/readonly for update mode PR #105

### Fixed
- Fixed bug of delete with datetime in primarykey crashing PR #105

> - Fixed issue website crashing when opening generate an input html block for datetime (#104) PR #106

Reviewer: ```suggestion
- Fixed issue where website crashes when opening a filter card for datetime. (#104) PR #106
```
